corpus/existing_aliases/as_alias_example.clj

```diff
@@ -0,0 +1,4 @@
+(ns as-alias-example
+  (:require [baz.qux :as-alias q]))
+
+(identity ::baz.qux/some-fn)
```

Comment:
This example isn't runnable in Clojure so maybe it's a different category than this error.

Reply:
Github is slow and doesn't let me edit my message. This error -> this linter.

Reply:
Do you mean Clojure 1.11? I didn't think about that. Could we check the clojure version and only warn if 1.11 or higher?

Reply:
It also doesn't work in any other clojure version if you just use `:as`. You cannot use auto-resolved keywords with anything else than an alias.

Reply:
Oh, I didn't know that.

Selfishly, I'd really like this one cuz I work on a Clojure-only codebase. Can we limit it to  just Clojure? Or would you prefer to have a separate related linter that only works on Clojure?

Reply:
Rereading this, seems I have fundamentally misunderstood the issue you raised. I've removed the whole test and removed all mention of keywords from the changelog and docs.